docs/git-cheat-detailed.md

```diff
@@ -0,0 +1,54 @@
+# Git Cheat Detailed
+
+See also
+[fixing commits](https://github.com/zulip/zulip-gci/blob/master/docs/fixing-commits.md)
```

Comment:
This is pointing to the old GCI docs. Ideally we would use a Markdown relative link, although I'm not sure if that's possible here.

In any case, this should point to the main repo stuff.